src/SearchResults.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect } from "react";
+import CustomerProfile from "./CustomerProfile";
+import moment from "moment";
+
+const SearchResults = (props) => {
+  const [selectedRows, setSelectedRows] = useState([]);
+  const [userID, setUserID] = useState("");
+  const [userData, setUserData] = useState("");
+
+  const clickHandler = (newID) => {
+    if (selectedRows.includes(newID)) {
+      setSelectedRows(selectedRows.filter((id) => id !== newID));
+    } else {
+      setSelectedRows([...selectedRows, newID]);
+    }
+  };
+
+  const userIDHandler = (userID) => {
+    return setUserID(userID);
+  };
+
+  useEffect(() => {
+    fetch(`https://cyf-react.glitch.me/customers/${userID}`)
+      .then((res) => res.json())
+      .then((data) => setUserData(data));
+  }, [userID]);
+
+  return (
+    <React.Fragment>
+      <table className="tabel">
+        <thead>
+          <tr>
+            <th>ID</th>
+            <th>Title</th>
+            <th>First Name</th>
+            <th>Surname</th>
+            <th>Email</th>
+            <th>Room ID</th>
+            <th>Check In Date</th>
+            <th>Check Out Date</th>
+            <th>Number of Nights</th>
+          </tr>
+        </thead>
+        <tbody>
+          {props.results.map((person) => {
+            const startDate = moment(person.checkInDate);
```

Comment:
For this it might be best to have the calculation above the jsx so that the logic and jsx is seprate